python/tests/test_drawing.py

```diff
@@ -252,6 +252,10 @@ def verify(self, tree):
 
         m1 = drawing.get_left_neighbour(tree, "minlex_postorder")
         m2 = get_left_neighbour(tree, "minlex_postorder")
+        # The following should work, but doesn't
+        # np.testing.assert_array_equal(m1, m2)
```

Comment:
This should work (m1 == m2), but the closest_left_node comparison won't work because that's based on the tree order.